web/themes/custom/pineapple/sass/components/paragraphs/_accordion.scss

```diff
@@ -0,0 +1,47 @@
+.accordion-title {
+  background-color: $primary;
+  color: $white;
```

Comment:
I think these colours are a bit strong, upon adding the accordion to the homepage for example i just felt that because of it using the full black it drew all the attention to it because it's right in the middle with the footer and nav using this same colour scheme of black and white and it almost felt out of place In my opinion. Consider using the light grey and black text to make it more consistent with the other components on the site such as your forms and cards, if you do decide to go for this i think making the text colour white on hover/focus also bodes well.

Reply:
Thank you. I absolutely love this suggestion. Looks a lot better.